File: data/readable_format/cpp/file051.cpp
#include <iostream>

// Example functionality to be tested
int get_number() {
    return 42;
}

// Simple test function
void test_get_number() {
    int result = get_number();
    if (result == 42) {
        std::cout << "Test passed: get_number() returned " << result << std::endl;
    } else {
        std::cout << "Test failed: get_number() returned " << result << " instead of 42" << std::endl;
    }
}

// Main function to run the test
int main() {
    std::cout << "Testing get_number function..." << std::endl;
    test_get_number();
    return 0;
}

File: data/readable_format/fortran/file051.f90
MODULE mod_behavior
  IMPLICIT NONE
CONTAINS
  FUNCTION get_number() RESULT(number)
    INTEGER :: number
    number = 42
  END FUNCTION get_number
END MODULE mod_behavior

PROGRAM test_mod_behavior
  USE mod_behavior
  IMPLICIT NONE
  INTEGER :: result

  result = get_number()

  WRITE(*,*) 'Testing get_number function...'
  IF (result == 42) THEN
    WRITE(*,*) 'Test passed: ', result
  ELSE
    WRITE(*,*) 'Test failed.'
  END IF
END PROGRAM test_mod_behavior